client/ayon_core/hosts/max/plugins/publish/validate_no_animation.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from pymxs import runtime as rt
+from ayon_core.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from ayon_core.hosts.max.api.action import SelectInvalidAction
+
+
+class ValidateNoAnimation(pyblish.api.InstancePlugin,
+                          OptionalPyblishPluginMixin):
+    """Validates No Animation
+
+    Ensure no keyframes on nodes in the Instance
```

Comment:
If the current logic remains where it checks only the controller's pos, rotate and scale I'd change this docstring to explain it only validates that.

However, as said below - I think it'd be better to just check for ANY animation on the node whatsoever instead of only transforms.